assets/lokomotive-kubernetes/azure/flatcar-linux/kubernetes/require.tf

```diff
@@ -9,7 +9,7 @@ provider "azurerm" {
 }
 
 provider "local" {
-  version = "~> 1.2"
+  version = "~> 1.4"
```

Comment:
Since those are not modules to direct use anymore, I think we can pin specific latest versions.